README.md

```diff
@@ -174,28 +174,28 @@ useminPrepare: {
 
 ### dest
 
-Type: 'string'  
```

Comment:
Why did you trim whitespace here? They're intentional line breaks and whitespace makes sense in markdown.